plugins/org.python.pydev.debug/src_console/org/python/pydev/debug/newconsole/PydevConsoleConstants.java

```diff
@@ -85,6 +85,17 @@
             { "OpenGL (glut)", "glut" },
             { "pyglet (pyglet)", "pyglet" } };
 
+    public static final String PYDEV_INTERPRETER_REPRESENTATION = "pydev";
+    public static final String PYTHON_INTERPRETER_REPRESENTATION = "python";
+    public static final String JYTHON_INTERPRETER_REPRESENTATION = "jython";
+    public static final String INTERACTIVE_CONSOLE_DEFAULT_INTERPRETER = "INTERACTIVE_CONSOLE_DEFAULT_INTERPRETER";
+    public static final String DEFAULT_INTERACTIVE_CONSOLE_DEFAULT_INTERPRETER = "none";
+    public static final String[][] ENTRIES_VALUES_INTERACTIVE_CONSOLE_DEFAULT_INTERPRETER = new String[][] {
+            { "Always ask for console type", "none" },
```

Comment:
The options should be the same options which are already in the `ChooseProcessTypeDialog`.

i.e.:

```
- Console for currently active editor
- Python console
- Jython console
- IronPython console
- Jython using VM running Eclipse console
- PyDev Debug Console
```

Note that some of those may be invalid if the user does not have the related VM configured (or is not under a debug session).

In case he selects an invalid one, this should be validated when the dialog is requested (so, if the selection is invalid he should open the dialog, otherwise, just do the default action).